pkg/jobs/resultcols.go

```diff
@@ -26,6 +26,17 @@ var BulkJobExecutionResultHeader = colinfo.ResultColumns{
 	{Name: "bytes", Typ: types.Int},
 }
 
+// OnlineRestoreJobExecutionResultHeader is the header for an online restore
+// job, which provides a header different from the usual bulk job execution
+var OnlineRestoreJobExecutionResultHeader = colinfo.ResultColumns{
+	{Name: "job_id", Typ: types.Int},
+	{Name: "status", Typ: types.String},
```

Comment:
nit: no need for the status column

Reply:
On a similar note, I didn't add it, but do you think there should be a `fraction_completed` column? Just something I noticed was being tested for in the regular restore in `backup_rand_test.go`.

Reply:
That column has always seemed pointless to me in a world where we only return one row after it is complete (or failed). I think that is a vestige from when we imagined we'd stream rows as it completed but then learned too many clients like to buffer rows and decided against it.

Reply:
Ah makes sense, I'll keep it excluded then. I did think it was a bit odd since we were always just returning a hardcoded 1 for it.